test/parallel/test-crypto-hmac.js

```diff
@@ -221,12 +221,12 @@ var rfc4231 = [
   }
 ];
 
-for (var i = 0, l = rfc4231.length; i < l; i++) {
-  for (var hash in rfc4231[i]['hmac']) {
-    var str = crypto.createHmac(hash, rfc4231[i].key);
+for (let i = 0, l = rfc4231.length; i < l; i++) {
+  for (const hash in rfc4231[i]['hmac']) {
```

Comment:
(In fact, I originally had it as `let`, but our linter complains that the value never changes and should be declared with `const` instead.)